src/components/buttons/Button/Button.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Icon } from "../../icons/Icon/Icon";
+
+import "./button.scss";
+
+/**
+ * Button
+ *
+ * All buttons used in the app
+ *
+ * @return {jsx}
+ */
+export const Button = ({
+  /* Add props here */
+  text,
+  iconName,
+  size,
+  color,
+  secondary,
+  ghost,
```

Comment:
@georgiganchev-7DIGIT I think instead of the properties "secondary", and "ghost" we should just have 1 property "type" which will accept one of the following: primary, secondary, or ghost. Then the color property will be accepting one of the following: purple or green. So in that way when you use the component you specify what it want it to be and also I think that the css will be better structured.